src/prefect_server/api/flow_groups.py

```diff
@@ -159,6 +159,33 @@ async def set_flow_group_labels(flow_group_id: str, labels: List[str] = None) ->
     return bool(result.affected_rows)
 
 
+@register_api("flow_groups.set_flow_group_run_config")
+async def set_flow_group_run_config(
+    flow_group_id: str, run_config: Dict[str, Any] = None
+) -> bool:
+    """
+    Sets run_config for a flow group.
+
+    Args:
+        - flow_group_id (str): the ID of the flow group to update
+        - run_config (dict, optional): a run-config override for a flow
+            group. Providing `None` defaults any previous flow group
+            `run_config` setting.
+
+    Returns:
+        - bool: whether setting `run_config` for the flow group was successful
+
+    Raises:
+        - ValueError: if flow group ID isn't provided
+    """
+    if not flow_group_id:
+        raise ValueError("Invalid flow group ID")
+    result = await models.FlowGroup.where(id=flow_group_id).update(
```

Comment:
Should we perform any validation on this input (e.g., attempt to deserialize it)?

Reply:
Cloud/server don't have models for run configs. We could add one, but it'd just check for the presence of a `type` and `labels` field (which is optional). Could go either way on this.

Reply:
Core technically has a serializer but I suppose that would destroy the benefits of using pydantic for compatibility purposes; we don't have to do anything about this now, just something to keep in the back of our minds